torchvision/io/image.py

```diff
@@ -191,40 +202,50 @@ def write_jpeg(input: torch.Tensor, filename: str, quality: int = 75):
     write_file(filename, output)
 
 
-def decode_image(input: torch.Tensor) -> torch.Tensor:
+def decode_image(input: torch.Tensor, channels: int = 0) -> torch.Tensor:
     """
     Detects whether an image is a JPEG or PNG and performs the appropriate
     operation to decode the image into a 3 dimensional RGB Tensor.
 
+    Optionally converts the image to the desired number of color channels.
     The values of the output tensor are uint8 between 0 and 255.
 
     Parameters
     ----------
     input: Tensor
         a one dimensional uint8 tensor containing the raw bytes of the
         PNG or JPEG image.
+    channels: int
+        the number of output channels or components of the decoded image.
+        JPEG and PNG images have different permitted values. See
+        `decode_jpeg()` and `decode_png()` for more information. Default: 0
```

Comment:
Can you mention here that 0 keeps the original number of channels? So that the users at least know what they get by default

Reply:
Good call.